index.html

```diff
@@ -0,0 +1,715 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/793d8924a3.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap">
+    <title>TNW</title>
+</head>
+
+<body>
+    <nav>
+        <ul>
+            <li>News</li>
+            <li>Events </li>
+            <li>Business </li>
+            <li>AMAs</li>
+            <li>Spaces</li>
+            <li>Terms & Conditions</li>
+        </ul>
+        <ul>
+            <li>About</li>
+            <li>Advertise</li>
+            <li>Jobs</li>
+            <li>Contact</li>
+        </ul>
+    </nav>
+    <div class="nav-2">
+        <div class="nav-2-left-holder">
+            <img class="hidden-1 nav-img" id="overwrite" alt="The Next Web"
+                src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+            <ul>
+                <li>LATEST</li>
+                <li>HARD FORK</li>
+                <li>PLUGGED</li>
+                <lI>README</lI>
+                <li>GROWTH QUARTERS</li>
+                <li>NEURAL</li>
+            </ul>
+        </div>
+        <div class="nav-2-right">
+            <i class="fas fa-search fa-flip-horizontal nav-2-icon"></i>
+            <i class="fas fa-bars nav-2-icon-2 hidden-2" id="hidden-2"></i>
+        </div>
+    </div>
+
+    <div class="top-grid">
+        <div class="top-grid-1">
+            <div class="child-sec">
+                <div class="head-text">
+                    <div class="logo-date">
+                        <div class="tmw-logo"><img src="./images/tnw.svg" alt="logo"></div>
+                    </div>
+                    <p class="current-date">Wednesday - 5, August 2020</p>
+                    <div class="headings">INSIGHTS</div>
+                    <div class="head-text-title">Kodak, now a pharma company, expects drugs to make up for 30-40% of its
+                        future business.</div>
+                    <div class="head-text-bottom">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago <i
+                            class="fas fa-share-alt"></i> 6</div>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] Now that you made the navbar fix, you have a problem with this section (it is under the navbar). Also re check the font size on the screen sizes in between the breakpoints. 

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89460579-e0588d80-d740-11ea-99b3-b25ed3772151.png) |![image](https://user-images.githubusercontent.com/61479648/89460671-0a11b480-d741-11ea-8309-2f688d1ca37a.png)  |

#### also in Mobile

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89460859-5e1c9900-d741-11ea-8853-f0c93bd62120.png) |![image](https://user-images.githubusercontent.com/61479648/89460877-6b398800-d741-11ea-9544-9b2394633f73.png)  |